python/rikai/contrib/torch/transforms/fasterrcnn_resnet50_fpn.py

```diff
@@ -38,29 +53,26 @@ def post_processing(options: Dict[str, Any]) -> Callable:
     def post_process_func(batch):
         results = []
         for predicts in batch:
-            predict_result = {
-                "boxes": [],
-                "label_ids": [],
-                "scores": [],
-            }
+            predict_result = []
             for box, label, score in zip(
                 predicts["boxes"].tolist(),
                 predicts["labels"].tolist(),
                 predicts["scores"].tolist(),
             ):
                 if score < min_score:
                     continue
-                predict_result["boxes"].append(box)
-                predict_result["label_ids"].append(label)
-                predict_result["scores"].append(score)
+                predict_result.append(
+                    {
+                        "box": Box2d(*box),
+                        "label_id": label,
+                        "score": score,
+                    }
+                )
 
             results.append(predict_result)
         return results
 
     return post_process_func
 
 
-OUTPUT_SCHEMA = (
-    "struct<boxes:array<array<float>>, scores:array<float>, "
-    "label_ids:array<int>>"
-)
+OUTPUT_SCHEMA = "array<struct<box:box2d, score:float, label_id:int>>"
```

Comment:
could be moved to a module named "detection"

this OUTPUT_SCHEMA is shared for this model and yolov5 and other detection models